spec/takeaway_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'takeaway'
+
+describe Takeaway do
+  
+  subject(:takeaway)    { described_class.new restaurant, order, messager }
+     let(:messager)     { double :messager    }
+     let(:restaurant)   { double :restaurant, menu: { "Pizza"=> 10.00, "Pasta"=> 8.00, "Milanese"=> 14.00} }
```

Comment:
Metrics/LineLength: Line is too long. [108/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)